molfeat/calc/_serializable_classes.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+from typing import Dict
+from typing import Any
+
+from rdkit.Chem import rdFingerprintGenerator
+
+SERIALIZABLE_CLASSES = {}
+
+
+def register_custom_class(cls: type):
```

Comment:
@isty2e  I believe there is also a bond invariant class, could you wrap that one too, just in case. After that, would just need for a green CI

Reply:
Added the class.